adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/model/DatasourceMetadata.java

```diff
@@ -0,0 +1,100 @@
+package org.jvalue.ods.adapterservice.datasource.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.persistence.Embeddable;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+import java.util.Objects;
+
+@Embeddable
+public class DatasourceMetadata {
```

Comment:
Pipeline and Datasource each have a metadata now. Probably makes sense, since authors may differ now.

Reply:
I also think that makes sence